packages/core/core/src/types.ts

```diff
@@ -290,6 +290,18 @@ export interface UseShippingErrors {
   load?: Error;
   save?: Error;
 }
+export interface UseBilling<BILLING, BILLING_PARAMS> {
+  error: ComputedProperty<UseBillingErrors>;
+  loading: ComputedProperty<boolean>;
+  billing: ComputedProperty<BILLING>;
+  load(): Promise<void>;
+  load(params: { customQuery?: CustomQuery }): Promise<void>;
+  save: (params: { params: BILLING_PARAMS; billingDetails: BILLING; customQuery?: CustomQuery }) => Promise<void>;
```

Comment:
why there are 2 versions of load but only one version of save?

Reply:
It is common approach in our app. Please look at UseCart, UseWishlist